src/server/firefox/ffNetworkManager.ts

```diff
@@ -209,14 +220,14 @@ class InterceptableRequest implements network.RouteDelegate {
     }) as any;
     if (!overrides.interceptResponse)
       return null;
-    return new InterceptedResponse(this.request, result.response.status, result.response.statusText, result.response.headers);
+    return new InterceptedResponse(this._request.request._finalRequest(), result.response.status, result.response.statusText, result.response.headers);
```

Comment:
I see we use `_finalRequest()` here, perhaps we can do the same in chromium?

Reply:
Done.